### ISSUE TITLE
How to use cifar-10 like single label dataset to evaluate this model?

### ISSUE THREAD

**Issue by Retrospection** [caoyue10/cvpr17-dvsq#6]
I want to use this model in some image retrieval tasks, but the nuswide_81 dataset is multilabel dataset, how can i use this in single label dataset image retrieval?

**Comment by bl0** [comment]
Hi, you can use this model in single label just same with the multilabel dataset without any change.

**Comment by bl0** [comment]
I will close this issue. Feel free to reopen it if u also have any question.

**Comment by Retrospection** [comment]
Thanks！

**Comment by Retrospection** [comment]
Excuse me, If I want to make this net to be a image feature extractor, that means, if I want to give an image and take a feature from this network without the whole bunch of config params, what should I do?

**Comment by bl0** [comment]
What do you mean by "feature"?  the 4096-dimensional feature(for fc7) or 64-dimensional feature(for fc8)?
If you are familiar with Tensorflow, simply run the graph and fetch the fc7(or fc8) as the feature(line 580 in net_val.py may help you).

**Comment by Retrospection** [comment]
yep, the "feature" I mean is just the fc8, but to use the net_val.py, I need to prepare the images in to dataset, make batchs to feed into the network, make whole bunch of config params. If I dont want to prepare such dataset, and dont want to prepare such config, what can I do? For example , If I just want to load the net param from .npy and simply use the skeleton of this network to make an inference to get the fc8, how can I achieve this? Thanks for your kindness

**Comment by bl0** [comment]
You can delete the useless part and only leave the skeleton. Then you need not prepare config and just need to:
1. load the pre-trained model into the model.
2. read the image from whatever place.
3. run the graph to get the feature.

**Comment by Retrospection** [comment]
Thanks for your great kindless

**Comment by bl0** [comment]
不客气😀
